src/components/ReleaseVersionsRangeFormControl.tsx

```diff
@@ -45,14 +45,12 @@ const ReleaseVersionsRangeFormControl = (props: StackProps) => {
   const [fromReleases, toReleases] = mapReleasesRange(releases)
 
   const selectPlaceholder =
-    Array.isArray(releases) && releases.length === 0
-      ? 'Releases not found'
-      : 'Choose a release'
+    Array.isArray(releases) && releases.length === 0 ? 'No releases found' : ''
```

Comment:
I noticed when there's no releases found, we show a error in the version select dropdown.

The dropdown box does not have enough space to explain why there could be no version, or what the user can do to troubleshoot:

- Check on GitHub if the repository really has no releases -> no problem.
- Check on GitHub and repository does have a release(s) -> report bug to Octoclairvoyant.

I think it would be nice to show a bigger popup box/modal, that shows a proper error message, and has helpful text on how to debug/report problems, etc.

Do you want an issue to discuss this idea in more detail?

Reply:
What about displaying the reason why the comparison is empty where we display the results?

Reply:
Elaborating a bit more on this: I prefer to avoid showing modals if the user workflow is not blocked anyhow.

For example, it makes sense to show a modal blocking the page when the GitHub requests limit is exceeded because the user can't use the website without authorization.

But in this case, it's just an explanation about why there are no results, so the user workflow is not blocked by anything, hence the preference on avoid a modal.

Reply:
> What about displaying the reason why the comparison is empty where we display the results?

I don't know if we have room for a big error message in that dropdown box? It just about manages to fit the current text in there... 😉 

> But in this case, it's just an explanation about why there are no results, so the user workflow is not blocked by anything, hence the preference on avoid a modal.

Ah okay, I had not considered that this is a non-blocking error, and so should not display a big fat modal... 😉

Reply:
> I don't know if we have room for a big error message in that dropdown box? It just about manages to fit the current text in there... 😉

I mean leaving the dropdown as it is and displaying the explanation you mentioned in the results area:
![imagen](https://user-images.githubusercontent.com/2677072/126895545-9c9c788f-68e7-4e2c-8db3-387d52926dbc.png)

Reply:
> > I don't know if we have room for a big error message in that dropdown box? It just about manages to fit the current text in there... 😉
> 
> I mean leaving the dropdown as it is and displaying the explanation you mentioned in the results area:
> ![imagen](https://user-images.githubusercontent.com/2677072/126895545-9c9c788f-68e7-4e2c-8db3-387d52926dbc.png)

I like your idea! 😄 It gives us the room to explain what went wrong, without blocking the user from re-trying with a different repo name.

Reply:
I've opened issue #282 to discuss this more if needed. 😉